lib/Elevate/OS/Base.pm

```diff
@@ -0,0 +1,79 @@
+package Elevate::OS::Base;
+
+=encoding utf-8
+
+=head1 NAME
+
+Elevate::OS::Base
+
+This is a base class currently used by Elevate::OS::*
+
+=cut
+
+use cPstrict;
+
+use Log::Log4perl qw(:easy);
+
+use Simple::Accessor qw{
+  ea_alias
+  elevate_rpm_url
+  default_upgrade_to
+  leapp_can_handle_epel
+  leapp_can_handle_imunify
+  leapp_can_handle_kernelcare
+  leapp_can_handle_python36
+  leapp_data_pkg
+  leapp_flag
+  name
+  pretty_name
+  upgrade_to
+};
+
+# NOTE: AlmaLinux_8 is just an alias for CentOS_8 so it really doesn't matter
+#       which of the two is used
+sub _build_ea_alias ($self) {
+    return 'CentOS_8';
+}
+
+sub _build_elevate_rpm_url ($self) {
+    die "subclass must implement elevate_rpm_url\n";
+}
+
+sub _build_default_upgrade_to ($self) {
```

Comment:
you'd have less code just with constants.

Reply:
This is done